org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -151,6 +153,19 @@ public static File getGradleHomeFile(Map<String, String> env, Properties sysprop
 		return null;
 	}
 
+	public static File getGradleUserHomeFile() {
+		if (JavaLanguageServerPlugin.getPreferencesManager() != null && JavaLanguageServerPlugin.getPreferencesManager().getPreferences().getGradleUserHome() != null) {
```

Comment:
should check getGradleUserHome()  is not blank

Reply:
Done.